app/controllers/foreman_tasks/tasks_controller.rb

```diff
@@ -99,7 +99,8 @@ def resource_class
 
     def respond_with_tasks(scope)
       @tasks = filter(scope, paginate: false)
-      csv_response(@tasks, [:id, :action, :state, :result, 'started_at.in_time_zone', 'ended_at.in_time_zone', :username], ['Id', 'Action', 'State', 'Result', 'Started At', 'Ended At', 'User'])
+      @tasks = @tasks.merge(ForemanTasks::Task.with_duration)
```

Comment:
You should be able to simplify this with `@tasks = filter(scope, paginate: false).with_duration`